libs/keyutils/Makefile

```diff
@@ -6,26 +6,32 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=keyutils
-PKG_VERSION:=1.5.10
+PKG_VERSION:=1.6
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://people.redhat.com/dhowells/keyutils/
```

Comment:
I see below you have changed the URL for HTTPS, but you can change it here as well, right?